CharlieBackend.Api/CharlieBackend.Api.csproj

```diff
@@ -23,15 +28,18 @@
     <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="3.1.3" />
     <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" Version="3.1.1" />
     <PackageReference Include="Serilog.AspNetCore" Version="3.4.0" />
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.5.1" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.6.3" />
     
     <PackageReference Include="DotNetEnv" Version="1.4.0" />
     
+    <PackageReference Include="Swashbuckle.AspNetCore.Filters" Version="6.0.1" />
+    
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\CharlieBackend.Business\CharlieBackend.Business.csproj" />
     <ProjectReference Include="..\CharlieBackend.Core\CharlieBackend.Core.csproj" />
+    <ProjectReference Include="..\CharlieBackend.Library\CharlieBackend.Library.csproj" />
```

Comment:
Why do we need separate project for examples? Also it is pretty generic name for swagger-specific classes. Please remove this project and place examples folder in CharlieBackend.Api.

Reply:
This was recommendation of Mentor on 20.11 meeting.

Ok.